tests/unit/test_backup.py

```diff
@@ -724,15 +729,22 @@ def test_restore_success(self):
             expire_time=timestamp,
         )
 
-        future = backup.restore(self.TABLE_ID)
+        future = backup.restore(self.TABLE_ID, instance_id)
```

Comment:
can we make this a new test instead of changing the existing?

Reply:
That's basically the idea. In order to avoid code repetition I created this common helper to be called by two separate tests (L743-L747):
```
    def test_restore_default(self):
        self._restore_helper()

    def test_restore_to_another_instance(self):
        self._restore_helper(self.ALT_INSTANCE, self.ALT_INSTANCE_NAME)
```
Would you still prefer having two fully coded, nearly identical tests instead?

Reply:
I would like one test that calls `restore` as is, and one test that calls restore passing in an instance id. If those are both covered, I am good 👍